lib/tasks/backfill_github_cached_fields_for_user.rake

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+task backfill_github_cached_fields_for_user: :environment do
+  puts "Backfilling GitHub-cached fields for Users"
+
+  BATCH_SIZE = 500
+
+  users_updated = 0
+  batch_num = 1
+
+  users_to_update = User.where(github_login: nil)
+  puts "There are #{users_to_update.count} users to update."
+
+  users_to_update.find_in_batches(batch_size: BATCH_SIZE) do |users|
+    puts "Updating batch #{batch_num}, starting with user ID #{users.first.id}"
+
+    users_updated += users.length
+    batch_num += 1
```

Comment:
Shouldn't we update these numbers _after_ the logins are fetched? If anything went wrong in our user loop these numbers would become inaccurate.